libdnf/goal/Goal.cpp

```diff
@@ -815,6 +815,7 @@ Goal::jobLength()
 bool
 Goal::run(DnfGoalActions flags)
 {
+    pImpl->run_flags = static_cast<DnfGoalActions>(flags);
```

Comment:
Why are you casting type to the same type?
Simply: `pImpl->run_flags = flags;`

Reply:
Fixed.